jwt_lib.h

```diff
@@ -0,0 +1,8 @@
+#ifndef JWT_H
+#define JWT_H
+
```

Comment:
chcelo by to trochu dokumentácie, pre inšpiráciu viď doxygen style dokumentáciu ktorú mám v newAuth knižnici